packages/json/src/lib/JSONProvider.ts

```diff
@@ -22,30 +19,36 @@ import {
   MathOperator,
   Method,
   Payloads,
-  PROPERTY_NOT_FOUND,
-  resolveCommonIdentifier,
-  setProperty
-} from '@joshdb/core';
+  resolveCommonIdentifier
+} from '@joshdb/provider';
 import { isPrimitive } from '@sapphire/utilities';
+import { resolve } from 'node:path';
+import { deleteProperty, getProperty, hasProperty, PROPERTY_NOT_FOUND, setProperty } from 'property-helpers';
+import { ChunkIndexFile } from './chunk-files/ChunkIndexFile';
 import { ChunkHandler } from './ChunkHandler';
 import type { File } from './File';
 
 export class JSONProvider<StoredValue = unknown> extends JoshProvider<StoredValue> {
   public declare options: JSONProvider.Options;
 
+  public get version(): JoshProvider.Semver {
+    return process.env.NODE_ENV === 'test' ? { major: 2, minor: 0, patch: 0 } : this.resolveVersion('[VI]{version}[/VI]');
```

Comment:
Fixed in the other providers but the ``NODE_ENV`` needs to be explained

Reply:
Jest sets the `NODE_ENV` environmental variable before tests are run. So, since we can't inject the version without rollup, we check if we're in tests here.

Reply:
Is it required for tests?

Reply:
For tests to run properly yes.

Reply:
Hm- interesting, tests work for mongo and redis without that env check

Reply:
Oh, nevermind then, I forgot `@joshdb/provider` doesn't have tests for migrations..